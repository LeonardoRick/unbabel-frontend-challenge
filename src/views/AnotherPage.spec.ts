import AnotherPageVue from '@/views/AnotherPage.vue';
import { mount } from '@vue/test-utils';

describe('AnotherPage', () => {
    it('should mount', () => {
        expect(mount(AnotherPageVue).html()).toMatchSnapshot();
    });
});
