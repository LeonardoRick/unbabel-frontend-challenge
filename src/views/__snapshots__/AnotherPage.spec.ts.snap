// Vitest Snapshot v1

exports[`AnotherPage > should mount 1`] = `
"<div class=\\"about\\">
  <h1>This is another page</h1>
  <h2>This page is just to show the router working properly</h2>
</div>"
`;
